bridge/github/config.go

```diff
@@ -205,11 +205,32 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("The remotes are:")
+
+		repos, nil := repo.ListRemotes()
+
+		var listOfRemotes []string
+		for i := 0; i < len(repos); i++ {
+			remoteUrls := strings.Split(strings.Split(repos[i], "	")[1], " ")
+			listOfRemotes = append(listOfRemotes, remoteUrls[0])
+			fmt.Printf("[%d]: %s\n", i+1, listOfRemotes[i])
+		}
+		fmt.Printf("[0]: Another project\n")
+
+		var choice int
+		_, err := fmt.Scanf("%d", &choice)
+
+		var line string
+		if choice == 0 {
+			fmt.Print("Github project URL: ")
+			line, err = bufio.NewReader(os.Stdin).ReadString('\n')
+		} else {
+			line = listOfRemotes[choice-1]
```

Comment:
This is not working (you should run the project and see if it works). Here, you need to extract the `projectName` and `projectUser` from the remote. Maybe do something similar as `splitURL` with a regex ?